CHANGELOG.md

```diff
@@ -1,5 +1,84 @@
 # Changelog
 
+## 💬 1.7.0
+
+- ### Features
+
+  - **Reformat config api-key output - [bradyjoslin], [issue/889] [pull/910]**
+
+    We care a lot about our error output. This PR from community contributor @bradyjoslin pretties up the output from `wrangler config` to match when you pass the `--api-key` flag. Thanks for the PR @bradyjoslin!
```

Comment:
nit: generally we put the contributor's name just at the top. open to having it in the changelog itself but we should change `@bradyjoslin` to `[bradyjoslin]` so the link works if we want to keep it there

Reply:
also when i've been writing these i've been trying to avoid "this PR" verbiage since they are _all_ PRs, but that might just be me